Intersect.Client/Entities/Entity.cs

```diff
@@ -1386,6 +1386,147 @@ public virtual void DrawName(Color textColor, Color borderColor = null, Color ba
+                        y = nameVertPos + nameSize.Y + 2;
+
+                        break;
+                    case NpcTagPos.Prefix:
+                        // Position the tag as prefix (2 pixels left from the name label).
+                        x = nameCentHorPos - (nameSize.X / 2) - tagTexture.GetWidth() - 6;
+                        y = nameVertPos + (nameSize.Y / 2) - (tagTexture.GetHeight() / 2);
+
+                        break;
+                    case NpcTagPos.Suffix:
+                        // Position the tag as suffix (2 pixels right from the name label).
+                        x = nameCentHorPos + (nameSize.X / 2) + 6;
+                        y = nameVertPos + (nameSize.Y / 2) - (tagTexture.GetHeight() / 2);
+
+                        break;
+                }
+                // And finally, we draw the tag.
+                if (tagTexture != null)
+                {
+                    Graphics.DrawGameTexture(tagTexture, x, y);
+                }
+            }
+            // Player Tags
+            if (Options.Player.ShowTags && this is Player && customPlayerTag)
+            {
+                // Custom Player Tag.
+                tagFileName = $@"Player_{entityName}.png";
+                // Now that we have the name of the sprite file, we load it as a texture.
+                GameTexture tagTexture = Globals.ContentManager.GetTexture(GameContentManager.TextureType.Tag, tagFileName);
+                // Before we draw the sprite, lets have it's position set.
+                switch (playerTagPos)
+                {
+                    case PlayerTagPos.Above:
+                    default:
+                        // Position the tag 2 pixels above the name label.
+                        x = nameCentHorPos - (tagTexture.GetWidth() / 2);
+                        y = nameVertPos - tagTexture.GetHeight() - 2;
+
+                        break;
+                    case PlayerTagPos.Under:
+                        // Position the tag 2 pixels under the name label.
+                        x = nameCentHorPos - (tagTexture.GetWidth() / 2);
+                        y = nameVertPos + nameSize.Y + 2;
+
+                        break;
+                    case PlayerTagPos.Prefix:
+                        // Position the tag as prefix (2 pixels left from the name label).
+                        x = nameCentHorPos - (nameSize.X / 2) - tagTexture.GetWidth() - 6;
+                        y = nameVertPos + (nameSize.Y / 2) - (tagTexture.GetHeight() / 2);
+
+                        break;
+                    case PlayerTagPos.Suffix:
+                        // Position the tag as suffix (2 pixels right from the name label).
+                        x = nameCentHorPos + (nameSize.X / 2) + 6;
+                        y = nameVertPos + (nameSize.Y / 2) - (tagTexture.GetHeight() / 2);
+
+                        break;
+                }
+                // And finally, we draw the tag.
+                if (tagTexture != null)
```

Comment:
Same null check problem I mentioned above for NPCs.

Reply:
👍🏽 

```
                // Now that we have the name of the sprite file, we load it as a texture.
                var tagTexture = Globals.ContentManager.GetTexture(GameContentManager.TextureType.Tag, tagFileName);
                // If the texture is null, we do nothing.
                if (tagTexture == null)
                {
                    return;
                }
                // Before we draw the sprite, lets have it's position set.
                switch (tagPos)
                    {
```